rbac/management/principal/cleaner.py

```diff
@@ -212,7 +213,14 @@ def process_umb_event(frame, umb_client: Stomp, bootstrap_service: TenantBootstr
             # If the setting is enabled, process all users.
             if not user.is_active or settings.PRINCIPAL_CLEANUP_UPDATE_ENABLED_UMB:
                 # If Tenant is not already ready, don't ready it
-                bootstrap_service.update_user(user, ready_tenant=False)
+                try:
+                    bootstrap_service.update_user(user, ready_tenant=False)
+                except ValueError as e:
```

Comment:
Especially with `nack` I think it is safer to broaden this, since the nack should trigger a limited number of retries. This depends on the policy of the broker, but it can be configured to a desirable behavior. For example in our case this should retry ~6 times before sending to a DLQ and unblocking the head of the queue.